plugins/modules/ntnx_ndb_databases.py

```diff
@@ -1499,6 +1499,12 @@ def delete_db_servers(module, result, database_info):
         spec = db_servers.get_default_delete_spec(
             delete=module.params.get("delete_db_server_vms", False)
         )
+
+        if module.check_mode:
```

Comment:
Test missing

Reply:
Why do we have check_sum for delete_db_servers, as we have it for delete_instance